k8s/live/azure-dev/postgresql-deployment.yaml

```
@@ -0,0 +1,68 @@
+      labels:
+        app: gridcapa
+        component: postgresql
+      annotations:
+        "cluster-autoscaler.kubernetes.io/safe-to-evict": "false"
+    spec:
+      containers:
+      - name: main
+        image: docker.io/bitnami/postgresql:14.6.0
+        imagePullPolicy: IfNotPresent
+        ports:
+          - containerPort: 5432
```

yamllint (1.35.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)